docs/xdr/FAQ/Assets_qa.md

```diff
@@ -0,0 +1,50 @@
+## How to get the source of a generated asset?
```

Comment:
Some remarks about the wordings "generated", "detected", etc.
The official wording for a discovered asset is **discovered**. I think it is important to avoid mixing this concept with *e.g.*, the notion of "detected" which should bring ambiguity *wrt* the detection workflow (or worse, the asset matching step in the events, which is different from discovering a new asset).

So I would replace "generated" with "discovered" everywhere we refer to an asset automatically created by the assetdiscovery feature